src/chrome/content/rules/nytimes3xbfgragh.onion.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+	For more NY Times ruleset, see NYTimes.xml
+
+	Still under construction:
+		myaccount.nytimes3xbfgragh.onion
+
+	Different content http/https:
+		lens.blogs.nytimes3xbfgragh.onion
+		samizdat-graphql.nytimes3xbfgragh.onion
+
+-->
+<ruleset name="nytimes3xbfgragh.onion">
+
+	<target host="nytimes3xbfgragh.onion" />
+	<target host="www.nytimes3xbfgragh.onion" />
+	<target host="a.nytimes3xbfgragh.onion" />
+		<test url="http://a.nytimes3xbfgragh.onion/svc/nyt/data-layer" />
+	<target host="*.blogs.nytimes3xbfgragh.onion" />
+		<test url="http://lens.blogs.nytimes3xbfgragh.onion/" />
+		<test url="http://afterdeadline.blogs.nytimes3xbfgragh.onion/" />
+		<test url="http://open.blogs.nytimes3xbfgragh.onion/" />
+	<target host="cn.nytimes3xbfgragh.onion" />
+	<target host="et.nytimes3xbfgragh.onion" />
+	<target host="global.nytimes3xbfgragh.onion" />
+	<target host="graphics.nytimes3xbfgragh.onion" />
+		<test url="http://graphics.nytimes3xbfgragh.onion/subscriptions/components/img/icons/icons_gift.png" />
+	<target host="mobile.nytimes3xbfgragh.onion" />
```

Comment:
```
<target host="samizdat-graphql.nytimes3xbfgragh.onion" />
<test url="http://samizdat-graphql.nytimes3xbfgragh.onion/graphql/v2" />
```

Reply:
Still different content http/https

Reply:
But nytimes3xbfgragh.onion makes use of the https:// version, search its source code for `samizdat`